requirements/celery50.txt

```diff
@@ -1,42 +1,9 @@
-#
```

Comment:
I'm curious why this file got so aggressively truncated. Make upgrade does still have this pip compile command.